state-synchronizer/src/executor_proxy.rs

```diff
@@ -219,7 +226,12 @@ impl ExecutorProxyTrait for ExecutorProxy {
             if !changed_configs.is_disjoint(&subscribed_items.configs)
                 || !event_keys.is_disjoint(&subscribed_items.events)
             {
-                subscription.publish(new_configs.clone())?;
+                if let Err(e) = subscription.publish(new_configs.clone()) {
+                    error!(
+                        "[state sync] failed to publish individual reconfig subscription: {}",
+                        e
+                    );
```

Comment:
can we print the subscription? can we make folks specify a name for each subscription?